config/config-istio.yaml

```diff
@@ -18,5 +18,11 @@ metadata:
   name: config-istio
   namespace: knative-serving
 data:
-  # Gateway used for knative traffic. Default to istio-ingressgateway.
-  ingress-gateway: "istio-ingressgateway.istio-system.svc.cluster.local"
+  # TODO(nghia): Extract the .svc.cluster.local suffix into its own config.
+
+  # ingress-gateway provides a semi-colon separated list of Istio
+  # Gateway that we should expose ClusterIngresses to, each having the form
+  # ${gatewayName}:${gatewayService}, where ${gatewayName} is the name of
+  # the Gateway, and ${gatewayService} is the URL of the service backing
+  # that gateway.
+  ingress-gateway: "knative-ingress-gateway:istio-ingressgateway.istio-system.svc.cluster.local;knative-shared-gateway:knative-ingressgateway.istio-system.svc.cluster.local"
```

Comment:
let's use a wildcard on keys, like this:
https://github.com/knative/serving/blob/0e7cbf395fe93d88ce74b93c23a615b2cede538b/config/config-logging.yaml#L48-L52

`*-gateway:` in `config-istio.yaml` seems like a safe compromise :)

Reply:
Since I don't want the Istio Gateway CRD to be named with -gateway suffix all the time, I opted to use a "gateway." prefix instead.